components/fieldgroup/index.css

```diff
@@ -27,9 +27,7 @@ governing permissions and limitations under the License.
 /* field group */
 .spectrum-FieldGroup {
   display: flex;
-  align-items: top;
```

Comment:
`align-items: top` is not valid CSS (it would need to be start or we would need to use `justify-content` here) so this property was not even applying, so I removed it. I can add it back in and change it to the proper value though if you'd like

Reply:
Ahhh right right. Now that you mention this, I've actually removed the same typo myself before!